### PR TITLE
#1240 Create Open file modal window 

### PR DESCRIPTION
**Pull request by ElenaOdnoshivkina**

### CODE REVIEW

File: packages/ketcher-polymer-editor-react/src/components/shared/actionButton/ActionButton.tsx [hunk 1]
```diff
@@ -38,24 +39,63 @@ const Button = styled(MuiButton)(({ theme }) => ({
...
+    border: `1px solid ${theme.ketcher.color.button.secondary.disabled}`,
+    color: theme.ketcher.color.button.secondary.disabled
+  },
+
+  '&:clicked': {
+    border: `1px solid ${theme.ketcher.color.button.secondary.clicked}`,
+    color: theme.ketcher.color.button.secondary.clicked
+  }
+}))
+
```
Comment (ViktoriiaFedotkina) [inline packages/ketcher-polymer-editor-react/src/components/shared/actionButton/ActionButton.tsx:69]: you can add some baseStyle object and move identical css props there

File: packages/ketcher-polymer-editor-react/src/components/modal/Open/OpenContainer.tsx [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ ***************************************************************************/
+import { Open } from './Open'
+
+const OpenContainer = ({ isModalOpen, onClose }) => {
```
Comment (ViktoriiaFedotkina) [inline packages/ketcher-polymer-editor-react/src/components/modal/Open/OpenContainer.tsx:18]: this container can be removed

**Review by ViktoriiaFedotkina — APPROVED** [review]

File: packages/ketcher-polymer-editor-react/src/Editor.tsx [hunk 3]
```diff
@@ -111,6 +101,9 @@ function MenuComponent() {
 
   const menuItemChanged = (name) => {
     dispatch(selectTool(name))
+    if (modalComponentList[name]) {
+      dispatch(openModal(name))
+    }
   }
```
Comment (karen-sarkisyan) [inline packages/ketcher-polymer-editor-react/src/Editor.tsx:107]: I don't think it's a good idea. For one, it sets active tool to 'Open', but there's no such tool. We should have some separation of this logic, and it probably should be handled inside of `Editor`.

**Review by karen-sarkisyan — CHANGES_REQUESTED** [review]

File: packages/ketcher-polymer-editor-react/src/components/modal/Open/TextEditor/TextEditor.tsx [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+const StyledTextarea = styled.textarea`
+  width: 405px;
+  min-width: 32em;
+  min-height: 23em;
+  overflow: auto;
+  white-space: pre;
+  resize: both;
+`
+
+export const TextEditor = ({ structStr, inputHandler }: TextEditorProps) => {
```
Comment (karen-sarkisyan) [inline packages/ketcher-polymer-editor-react/src/components/modal/Open/TextEditor/TextEditor.tsx:32]: Maybe we should have a shared styled textarea component, same one we create for Save modal? https://github.com/epam/ketcher/pull/1246/files#diff-29ff483d977e475e9b6e4578a4b165776d0c3f268c75f2d8a5cc6140b78b0b32

**Review by karen-sarkisyan — CHANGES_REQUESTED** [review]

File: packages/ketcher-polymer-editor-react/package.json [hunk 5]
```diff
@@ -50,6 +50,7 @@
     "@emotion/styled": "^11.6.0",
     "@mui/material": "^5.2.4",
     "@reduxjs/toolkit": "^1.7.1",
+    "@testing-library/dom": "^8.11.3",
```
Comment (karen-sarkisyan) [inline packages/ketcher-polymer-editor-react/package.json:53]: This should be in devDependencies :)

**Review by karen-sarkisyan — CHANGES_REQUESTED** [review]

**Review by karen-sarkisyan — APPROVED** [review]

**Pull request merged**